v0/destinations/zendesk/transform.js

```diff
@@ -344,6 +344,12 @@ async function processIdentify(message, destinationConfig, headers) {
     destinationConfig,
     "identify"
   );
+  if (!payload.user.name) {
+    throw new CustomError(
+      "[Zendesk] Missing required value from [traits.name]",
+      400
+    );
+  }
```

Comment:
We shouldn't be fixing it here. Seems like it needs to be fixed in the common utils.